### PR TITLE
adding accessory view to the right of Message Bar

### PR DESCRIPTION
**Pull request by okan-erdogan**
messagebar

### CODE REVIEW

File: SwiftMessageBar/Message.swift [hunk 1]
```diff
@@ -23,13 +19,25 @@ final class Message: UIView {
...
+  private var accessoryView: UIView?
+
+  public lazy var contentStackView: UIStackView = {
+    let contentView = UIStackView(frame: bounds)
+    contentView.layoutMargins = UIEdgeInsets(top: 10, left: 10, bottom: 10, right: 10)
+    contentView.isLayoutMarginsRelativeArrangement = true
+    contentView.axis = .horizontal
+    contentView.spacing = 10
+
+    usesAutoLayout(true)
```
Comment (JanGorman) [inline SwiftMessageBar/Message.swift]: Which view is this call to `usesAutoLayout` referring to?

File: SwiftMessageBar/SwiftMessageBar.swift [hunk 2]
```diff
@@ -207,9 +207,10 @@ public final class SwiftMessageBar {
   public static func showMessage(withTitle title: String? = nil, message: String? = nil, type: MessageType,
                                  duration: TimeInterval = 3, dismiss: Bool = true,
                                  languageDirection: NSLocale.LanguageDirection = .unknown,
+                                 accessoryView: UIView? = nil,
                                  callback: Callback? = nil) -> UUID {
     return sharedMessageBar.showMessage(withTitle: title, message: message, type: type, duration: duration,
-                                        dismiss: dismiss, languageDirection: languageDirection, callback: callback)
+                                        dismiss: dismiss, languageDirection: languageDirection, accessoryView: accessoryView, callback: callback)
```
Comment (JanGorman) [inline SwiftMessageBar/SwiftMessageBar.swift:214]: Can you add a comment to the parameters list for `accessoryView` please

File: SwiftMessageBar/SwiftMessageBar.swift [hunk 3]
```diff
@@ -228,12 +229,13 @@ public final class SwiftMessageBar {
...
                           duration: TimeInterval = 3, dismiss: Bool = true,
                           languageDirection: NSLocale.LanguageDirection = .unknown,
+                          accessoryView: UIView? = nil,
                           callback: Callback? = nil) -> UUID {
     let message = Message(type: type, title: title, message: message,
                           backgroundColor: type.backgroundColor(fromConfig: config), titleFontColor: config.titleColor,
                           messageFontColor: config.messageColor, icon: type.image(fromConfig: config), duration: duration,
                           dismiss: dismiss, callback: callback, languageDirection: languageDirection,
-                          titleFont: config.titleFont, messageFont: config.messageFont)
+                          titleFont: config.titleFont, messageFont: config.messageFont, accessoryView: accessoryView)
```
Comment (JanGorman) [inline SwiftMessageBar/SwiftMessageBar.swift:240]: And same here with the comment

**Review by JanGorman — COMMENTED** [review]
Godd stuff

File: SwiftMessageBar/SwiftMessageBar.swift (hunk 2 above)
Comment (okan-erdogan) [reply SwiftMessageBar/SwiftMessageBar.swift:214]: I added comment please review again.

File: SwiftMessageBar/Message.swift (hunk 1 above)
Comment (okan-erdogan) [reply SwiftMessageBar/Message.swift]: I removed it.

**Comment by Igor-Palaguta** [comment]
In README iOS version should be changed and Video probably should be updated

**Pull request merged**